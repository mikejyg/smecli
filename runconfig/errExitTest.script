localEcho
localEcho on
coe off
. errExit1
echo this should not happen in errExitTest!
