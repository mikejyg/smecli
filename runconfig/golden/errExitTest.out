> Task :compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :jar UP-TO-DATE
> Task :assemble UP-TO-DATE
> Task :compileTestJava UP-TO-DATE
> Task :processTestResources UP-TO-DATE
> Task :testClasses UP-TO-DATE
> Task :test UP-TO-DATE
> Task :check UP-TO-DATE
> Task :build UP-TO-DATE

BUILD SUCCESSFUL in 0s
5 actionable tasks: 5 up-to-date
> OK
off
> OK
on
> coe off
OK
off
> . errExit1
> echo errExit1
OK
errExit1
> source errExit2
> echo errExit2
OK
errExit2
> ebcd
INVALID_COMMAND
SCRIPT_ERROR_EXIT
